src/pip/_internal/pep425tags.py

```diff
@@ -333,11 +270,17 @@ def _mac_platforms(arch):
     match = _osx_arch_pat.match(arch)
     if match:
         name, major, minor, actual_arch = match.groups()
-        tpl = '{}_{}_%i_%s'.format(name, major)
-        arches = []
-        for m in reversed(range(int(minor) + 1)):
-            for a in get_darwin_arches(int(major), m, actual_arch):
-                arches.append(tpl % (m, a))
+        mac_version = (int(major), int(minor))
+        mac_arches = list(mac_platforms(mac_version, actual_arch))
```

Comment:
nit: the list call here is unnecessary.

Reply:
Thanks, I will inline it in the list comprehension.